styles/Events.module.scss

```diff
@@ -76,32 +83,36 @@
       transform-origin: bottom left;
     }
   }
-}
-
-
-
+  @media (max-width: 1024px) {
+    grid-template-columns: 190px 1fr;
+    width: 100%;
+  }
+  @media (max-width: 1600px) {
+    grid-template-columns: 150px 1fr;
+  }
 
-.event-image {
-  height: 100%;
-  width: 100%;
 }
 
 .event-header {
+  align-items: center;
+  display: flex;
   font-size: 1.3rem;
   font-weight: 500;
   grid-area: title;
+  justify-content: center;
   line-height: normal;
-  margin: auto;
-  text-align: left;
-  width: 300px;
+  margin-top: 1.25rem;
+  max-width: 90%;
 
   h2 {
     -webkit-box-orient: vertical;
+    /* stylelint-disable-next-line value-no-vendor-prefix */
+    display: -webkit-box;
```

Comment:
Why is this needed?

Reply:
I was checking in Mozilla firefox but this style is not present, seems like SASS removes that style while compiling

![image](https://user-images.githubusercontent.com/36519478/190417133-c0dd839a-a7ea-47a3-b9da-84e38c888312.png)

Reply:
In the docs it mention those are not handled internally by autoprefixer
https://github.com/postcss/autoprefixer
https://github.com/postcss/autoprefixer#how-to-work-with-legacy--webkit--only-code

Reply:
Please confirm

Reply:
Seems that Mozilla firefox compiles correctly `-webkit-box`